mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -2999,23 +3007,21 @@ convertOmpTargetData(Operation *op, llvm::IRBuilderBase &builder,
       [&moduleTranslation](
           llvm::OpenMPIRBuilder::DeviceInfoTy type,
           llvm::ArrayRef<BlockArgument> blockArgs,
-          llvm::OpenMPIRBuilder::MapValuesArrayTy &basePointers,
-          llvm::OpenMPIRBuilder::MapDeviceInfoArrayTy &devicePointers,
+          llvm::SmallVectorImpl<Value> &devicePtrVars, MapInfoData &mapInfoData,
```

Comment:
Nit: Since it's used for both `use_device_ptr` and `use_device_addr`, to avoid confusion.
```suggestion
          llvm::SmallVectorImpl<Value> &useDeviceVars, MapInfoData &mapInfoData,
```